capnpy/_hash.h

```diff
@@ -0,0 +1,19 @@
+#include <Python.h>
+
+// duck-typed _Py_HashSecret_t from CPython 2.7 (different from 3.5)
```

Comment:
The comment is correct, this is from 2.7
You are probably right about the longer comment. Will add proper explanation.